packages/api/cms-api/src/redirects/import-redirects.console.ts

```diff
@@ -44,7 +44,7 @@ export class ImportRedirectsConsole {
         let errors = 0;
 
         for (const row of rows) {
-            const node = await this.pageTreeService.createReadApi({ visibility: "all" }).getNodeByPath(row["target"]);
+            const node = await this.pageTreeService.createReadApi({ visibility: "all" }).getNodeByPath(row["target"], { scope: row["scope"] });
```

Comment:
This will fail for unscoped redirects: `Uncaught Error: Trying to query by not existing property Redirect.scope`. We need to move the scope check into an if like we do in L50-52.

Reply:
c8635f8a4baf44f548613e66927bdc570accd48c